index.js

```diff
@@ -9,7 +9,13 @@ const RequestData = require("./lib/txRequest/requestData")
 const version = require('./package.json').version;
 const contracts = require("./lib/build/ethereum-alarm-clock.json").version;
 
-module.exports = (web3) => {
+/**
+ * 
+ * @param {Web3} web3 The web3 object (required). 
+ * @param {Object} assetJSON The asset JSON object which contains the contracts names
+ * mapped to the addresses.
+ */
+module.exports = (web3, assetJSON) => {
```

Comment:
I'd initialize `assetJSON` to `null` to make it obvious that it's an optional parameter.